pkg/cloud/network_test.go

```diff
@@ -108,6 +122,10 @@ var _ = Describe("Network", func() {
 			ns.EXPECT().NewCreateNetworkParams(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).
 				Return(&cloudstack.CreateNetworkParams{})
 			ns.EXPECT().CreateNetwork(gomock.Any()).Return(&cloudstack.CreateNetworkResponse{Id: netId}, nil)
+			rs.EXPECT().NewListTagsParams().Return(&cloudstack.ListTagsParams{})
+			rs.EXPECT().ListTags(gomock.Any()).Return(&cloudstack.ListTagsResponse{}, nil)
+			rs.EXPECT().NewCreateTagsParams(gomock.Any(), gomock.Any(), gomock.Any()).Return(&cloudstack.CreateTagsParams{})
```

Comment:
Could use the param matchers I built here so that we're checking the params are reasonable. No biggie though.

Reply:
Argument matches on mock stubs can make tests difficult to debug.  If I were to check the args, I'd prefer to do it in a verification stub at the end of the test.  But it doesn't look like gomock provides that ability.  (FWIW, it was requested 3 years ago.  https://github.com/golang/mock/issues/262)

I can throw a few checks in there, anyway, for data that's shared by all the tests.  (I'm moving these lines into a BeforeEach.